apps/journeys-admin/src/components/Editor/ControlPanel/ColorDisplayIcon/ColorDisplayIcon.tsx

```diff
@@ -0,0 +1,52 @@
+import { ReactElement } from 'react'
+import Paper from '@mui/material/Paper'
+import Box from '@mui/material/Box'
+import { ThemeProvider } from '@core/shared/ui'
+import { TreeBlock, useEditor } from '@core/journeys/ui'
+import { GetJourney_journey_blocks_CardBlock as CardBlock } from '../../../../../__generated__/GetJourney'
+import {
+  TypographyColor,
+  ThemeName,
+  ThemeMode
+} from '../../../../../__generated__/globalTypes'
+
+interface ColorDisplayIconProps {
+  color: TypographyColor | null
+}
+
+export function ColorDisplayIcon({
+  color
+}: ColorDisplayIconProps): ReactElement {
+  const {
+    state: { selectedStep }
+  } = useEditor()
+
+  const card = selectedStep?.children.find(
+    (block) => block.__typename === 'CardBlock'
+  ) as TreeBlock<CardBlock> | undefined
+
+  return (
+    <Paper
+      sx={{
+        borderRadius: 1000
+      }}
+    >
+      <ThemeProvider
+        themeName={card?.themeName ?? ThemeName.base}
+        themeMode={card?.themeMode ?? ThemeMode.dark}
+        nested
+      >
+        <Box
+          data-testid="color-display-icon"
```

Comment:
```suggestion
          data-testid=`${color}-color-icon`
```
Make sure id's are unique